insights/parsers/swift_conf.py

```diff
@@ -0,0 +1,44 @@
+"""
+swift_conf - files under /etc/swift/
+====================================
+
+This module provides parsers for swift config files under /etc/swift directory.
+
+Class ``SwiftProxyServerConf`` parses the content of the ``/etc/swift/proxy-server.conf``.
```

Comment:
It would be better to shift line-7~35 into the docstring `SwiftProxyServerConf`, just like the `netstat.py`